Common/Orders/OrderEvent.cs

```diff
@@ -174,7 +182,7 @@ public override string ToString()
                 : string.Format("Time: {0} OrderID: {1} Symbol: {2} Status: {3} Quantity: {4} FillPrice: {5} {6}", UtcTime, OrderId, Symbol.Value, Status, FillQuantity, FillPrice.SmartRounding(), FillPriceCurrency);
 
             // attach the order fee so it ends up in logs properly
-            if (OrderFee != 0m) message += string.Format(" OrderFee: {0} {1}", OrderFee, CashBook.AccountCurrency);
+            if (OrderFee != 0m) message += string.Format(" OrderFee: {0} {1}", OrderFee, AccountCurrency);
```

Comment:
There's an interesting hidden issue here, and that is that the fee might _not_ be in units of the account currency. For example, it's very common on crypto exchanges to charge fees in units of the quote currency. So if I trade +10 units of LTCBTC, I'll get charged fees in BTC and not in USD. I think we should probably fix the order fees in a separate PR before this account currency work is completed. We can rephrase the order fee calculation methods to return a cash object which would contain the quantity and units of the fee, as well as a conversion rate to the account currency. This also precludes us from piping the value everywhere as a string and instead more accurately describes the fee.